tests/unit/test_client_options.py

```diff
@@ -145,9 +146,6 @@ def test_from_dict_bad_argument():
 
 def test_repr():
     options = client_options.ClientOptions(api_endpoint="foo.googleapis.com")
-
-    assert (
-        repr(options)
-        == "ClientOptions: {'api_endpoint': 'foo.googleapis.com', 'client_cert_source': None, 'client_encrypted_cert_source': None, 'api_key': None}"
-        or "ClientOptions: {'client_encrypted_cert_source': None, 'client_cert_source': None, 'api_endpoint': 'foo.googleapis.com', 'api_key': None}"
-    )
+    options_repr = repr(options)
+    assert match(r"ClientOptions:", options_repr)
+    assert match(r".*api_endpoint.*foo.googleapis.com", options_repr)
```

Comment:
```suggestion
    assert match(r".*'api_endpoint': 'foo.googleapis.com'.*", options_repr)
```

don't you think, so we make sure the value is paired with the right key?